ballerina/build.gradle

```diff
@@ -909,7 +909,10 @@ task testExamples() {
         'counter-metrics',
         'tracing',
         'openapi-to-ballerina',
-        'taint-checking' // Should not compile
+        'taint-checking', // Should not compile
+        'task-service-appointment',
+        'task-scheduler-appointment',
+        'user-defined-error'
```

Comment:
Lets open an issue to track this and tag the relevant people to fix this

Reply:
Added 
- https://github.com/ballerina-platform/ballerina-distribution/issues/924
- https://github.com/ballerina-platform/ballerina-distribution/issues/925